decidim-verifications/app/views/decidim/verifications/id_documents/authorizations/_form.html.erb

```diff
@@ -1,13 +1,9 @@
 <%= form_required_explanation %>
 
-<div class="card-section">
-  <div class="field">
-    <%= form.select :document_type, @form.document_types_for_select, prompt: true %>
-  </div>
+<div class="form__wrapper">
+  <%= form.select :document_type, @form.document_types_for_select, prompt: true %>
```

Comment:
Also to add a general note here that the form fields still look very disabled in their normal state (grey):
![Form fields](https://user-images.githubusercontent.com/864340/203937794-4ce4dc85-275c-4dcb-bdf0-00094b9b539c.png)

The form fields should look like the user can write into them.

Also, please double check the contrast requirements on these (according to WCAG criteria).

Reply:
We'll create a new issue to tackle this so it doesn't block this merge, which we should clear ASAP.

Reply:
(Please, when we encounter things that are design issues and that should be treated as structural rather than something specific from the given PR, as this issue, let's create issues so we can progress with the merges).